mqttshark

```diff
@@ -534,7 +546,7 @@ def print_publish(args, packet, mqtt):
         return
 
     try:
-        payload = mqtt['mqtt_mqtt_msg_text']
+        payload = bytearray.fromhex(mqtt['mqtt_mqtt_msg'].replace(":", "")).decode()
```

Comment:
Do you think we should use a separate function for decoding the MQTT message payload, in order to improve testing capabilities?